README.md

```diff
@@ -1 +1,129 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Hello Mircoverse <a name="about-project"></a>
+
+**Hello Mircoverse** is my first project as Micronaut which has a simple webpage. The project is done to master all the tools and best practices required to setup Microverse project repositories.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **proper_documentation**
+- [ ] **github_flow**
+- [ ] **linter_for_html&css**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/ANTOINE1128/Hello-Microverse.git)
+
```

Comment:
> * [ ]  [OPTIONAL] Please I suggest you remove the live demo link since  your project has not been deployed yet

- [ ] Like the previous code reviewer said, kindly do away with this as it will just cause noise in your README file as this is part of the requirements, it has just been made optional because the previous reviewer didn't talk about it 😀.